test/inductor/test_inplacing_pass.py

```diff
@@ -143,6 +155,125 @@ def f(x):
         # IF THIS NUMBER GOES TO ZERO, PLEASE FIND ANOTHER EXAMPLE
         self.assertEqual(num_reinplacing_failures(), 1)
 
+    def get_not_inplaced_count(self, graph):
+        counter = 0
+        for node in graph.nodes:
+            if node.target == torch.ops.higher_order.auto_functionalized:
```

Comment:
we should assert that the auto_functionalized node was found (otherwise, can erroneously report 0)